pkgs/build-support/build-fhs-userenv/env.nix

```diff
@@ -124,6 +124,24 @@ let
     paths = [ etcPkg ] ++ basePkgs ++ targetPaths;
     extraOutputsToInstall = [ "out" "lib" "bin" ] ++ extraOutputsToInstall;
     ignoreCollisions = true;
+    postBuild = ''
+      if [ -d  $out/share/gsettings-schemas/ ]; then
+          # Create the standard schemas directory
+          rm -rf $out/share/glib-2.0
+          mkdir -p $out/share/glib-2.0/schemas
+
+          # symlink any schema files to the standard schema directory
+          for d in $out/share/gsettings-schemas/*; do
+              # Force symlink, in case there are duplicates
+              ln -fs $d/glib-2.0/schemas/*.xml $out/share/glib-2.0/schemas
```

Comment:
Do we care about overrides?

Reply:
We probably should, thanks :+1: It should all work like in FHS environments.